src/wiki/templates/wiki/includes/render.html

```diff
@@ -5,13 +5,7 @@
 {% endaddtoblock %}
 
 <div class="wiki-article">
-  {% if not preview %}
-    {% if article.current_revision %}
-      {{ article.get_cached_content }}
-    {% endif %}
-  {% else %}
-    {{ content|default:"" }}
-  {% endif %}
```

Comment:
Seems correct that the above should better be replaced with something more understandable.

Did this code also cause a bug?

Reply:
yes, get_cached_content does not get 'user' argument with this code. Also wiki_tags:wiki_render() is probably better place that the template (at least it saves 4 lines of code)